applications/dashboard/views/profile/helper_functions.php

```diff
@@ -7,9 +7,11 @@
      * @return string
      */
     function userVerified($user) {
-        $userID = val('UserID', $user);
+        $userID = $user->UserID;
```

Comment:
It looks like this would've worked with arrays or objects. Now, it'll only work with objects. This is a globally-available function. That's going to break some backwards compatibility.